stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -98,6 +99,14 @@ protected void afterRun() {
     }
 
     /**
+     * Override this method if you need to execute code once after all workers have finished their run phase.
```

Comment:
Can you explain if this always is going to be called; e.g. if there was an exception? Or only if everything went fine.